src/main.rs

```diff
@@ -36,6 +36,8 @@ async fn main() {
 
     let file = if let Ok(f) = std::env::var("CONFIG_FILE") {
         f
+    } else if fs::metadata("/etc/rac/client.toml").is_ok() {
```

Comment:
I would rather use `Path::exists`

Reply:
I didn't know about `Path::exists`, and apparently it's just a Macro (or whatever name Rust uses) to `fs::metadata(...).is_ok()`.

https://github.com/rust-lang/rust/blob/1.32.0/src/libstd/path.rs#L2406-L2408

Any reason I don't know I should use `Path::exists`?

Reply:
Well it says ```If you want to
    /// check errors, call [fs::metadata].
```
and we're checking the `std::result` so I guess leaving like this is better.

Reply:
Alright, sure...

I changed it to https://doc.rust-lang.org/std/fs/fn.try_exists.html according to the docs, it's a bit better than the other two options. I unwraped the `Err(_)` return values as well because we don't treat it.